lib/events/filesessions/fileasync.go

```diff
@@ -637,9 +610,7 @@ func (u *Uploader) startUpload(ctx context.Context, fileName string) (err error)
 		return trace.ConvertSystemError(err)
 	}
 
-	u.wg.Add(1)
-	go func() {
-		defer u.wg.Done()
+	u.wg.Go(func() {
```

Comment:
This change needs to be reverted. branch/v18 is still using Go 1.24 which doesn't have waitgroup.Go :(.

Reply:
`synctest` was a pain too :sweat_smile: Ended up rolling those tests back to using fake clocks. I noticed we have a build tag for using `synctest`, but do those tests actually run anywhere?